hw/ip/edn/doc/theory_of_operation.md

```diff
@@ -62,38 +70,42 @@ To ensure that the most recently enabled EDN will get next priority for physical
+For [Top Earlgrey](../../../top_earlgrey/README.md), the first random values distributed from this EDN are therefore expected to be available roughly 2ms after reset.
+
 The `entropy_src` only creates one pre-FIPS seed, so any other EDNs must wait for their seeds to pass the full FIPS-recommended health checks.
-This means that each subsequent EDN must wait an additional 5ms before it can start distributing data.
+This means that for [Top Earlgrey](../../../top_earlgrey/README.md), each subsequent EDN must wait an additional 5ms before it can start distributing data.
 For instance, if there are three boot-time request mode EDN's in the system, the first will start distributing data 2ms after reset, the second will start distributing data 7ms after reset, and the third will start distributing data 12ms after reset.
 
 ### Auto Request Mode
 
-Before entering auto request mode, it is the responsibility of firmware to first generate an `instantiate` command for the EDN-associated instance via the [`SW_CMD_REQ`](registers.md#sw_cmd_req) register.
-The required `generate` and `reseed` commands must also be custom generated by firmware and loaded into the respective command replay FIFOs via the [`GENERATE_CMD`](registers.md#generate_cmd) and [`RESEED_CMD`](registers.md#reseed_cmd) registers.
+Before entering auto request mode, it is the responsibility of firmware to first load the required `generate` and `reseed` commands into the respective command replay FIFOs via the [`GENERATE_CMD`](registers.md#generate_cmd) and [`RESEED_CMD`](registers.md#reseed_cmd) registers.
 These `generate` commands will be issued as necessary to meet the bandwidth requirements of the endpoints.
-The `reseed` commands will be issued once every `MAX_NUM_REQS_BETWEEN_RESEEDS` generate requests.
-The reset value is `0`.
-This implies that no generate commands will be issued unless this value is changed by firmware.
+The `reseed` commands will be issued once every `MAX_NUM_REQS_BETWEEN_RESEEDS` `generate` requests.
+Before entering auto request mode, firmware must configure a non-zero value in the [`MAX_NUM_REQS_BETWEEN_RESEEDS`](registers.md#max-num-reqs-between-reseeds) register.
+The reset value is `0`, which means no `generate` commands are issued.
 For details on the options for application interface commands please see the [CSRNG IP Documentation](../../csrng/README.md).
-Once the CSRNG instance has been instantiated, and the `generate` and `reseed` commands have been loaded, auto request mode can be entered by programming the [`CTRL`](registers.md#ctrl) register with `EDN_ENABLE` and `AUTO_REQ_MODE` fields are enabled.
-Note that if BOOT_REQ_MODE is asserted the state machine will enter boot-time request mode, even if AUTO_REQ_MODE is asserted.
+Auto request mode can then be entered by programming the [`CTRL`](registers.md#ctrl) register with `EDN_ENABLE` and `AUTO_REQ_MODE` fields asserted.
```

Comment:
Nit: I know this was true before, but the text would probably be clearer if we added an article to give something like "... with the EDN_ENABLE and AUTO_REQ_MODE fields asserted".